tests/test_cli.py

```diff
@@ -237,14 +237,14 @@ def test_multiple_files(script_runner):
     assert "[6.574722, 51.434444, 4.3175, 53.217222]" in ret.stdout, "bboxes and time values of all files inside folder, are printed to console"
     assert "[292063.81225905, 5618144.09259115, 302531.3161606, 5631223.82854667]" in ret.stdout, "bboxes and time values of all files inside folder, are printed to console"
 
-@pytest.mark.skip(reason="director input not implemented yet")
 def test_folder(script_runner):
```

Comment:
What about a CLI test for a ZIP file?

Reply:
I'll include  a CLI test for ZIP files !